core/templates/dev/head/components/question-directives/question-player/question-player.constants.ts

```diff
@@ -43,4 +47,10 @@ oppia.constant(
 
 oppia.constant('VIEW_HINT_PENALTY', 0.1);
 
+oppia.constant('VIEW_HINT_PENALTY_FOR_MASTERY', 0.02);
+
+oppia.constant('VIEW_SOLUTION_PENALTY_FOR_MASTERY', 0.15);
```

Comment:
We do not need this constant, use the MAX_LOSS constant instead when user views solution

Reply:
Done.